typesense.org/pages/typesense-vs-algolia-vs-elasticsearch-vs-meilisearch.vue

```diff
@@ -632,15 +661,8 @@
                   <td>
                     No limitation, only constrained by available disk space
                   </td>
-                  <td class="text-danger">
-                    Slow indexing performance
-                    <a
-                      href="https://github.com/meilisearch/meilisearch/discussions/2131"
-                      target="_blank"
-                      class="text-danger"
-                      >reported</a
-                    >
-                    for over 10K records.
```

Comment:
Here is one of the many [users reporting](https://github.com/meilisearch/meilisearch/discussions/3450#discussioncomment-4862004) that he is using Meilisearch for 3x to 4x more than the limit you are talking about that is one year old.